RingPublishingGDPR/src/main/AndroidManifest.xml

```diff
@@ -6,7 +6,7 @@
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
 
     <application>
-        <activity android:name="com.ringpublishing.gdpr.RingPublishingGDPRActivity" android:theme="@style/GDPRStyle" />
+        <activity android:name="com.ringpublishing.gdpr.RingPublishingGDPRActivity" android:theme="@style/GDPRStyle" android:launchMode="singleTask" />
```

Comment:
a to jest kompatybilne wstecz? Jak ktoś to nagle dostanie to nic mu sie nie stanie?